### DISCUSSION TITLE
Test idea: service banners on the portal

### DISCUSSION THREAD

**Discussion by a-surfer** [UtrechtUniversity/yoda#176] [Ideas]
As a Yoda system administrator
I want to offer status updates and news on the Yoda Portal
So that I can announce maintenance, or report special situations directly to Yoda's users

**Comment by a-surfer** [comment]
@lwesterhof , I would like to mark this idea as an RDM-TEC feature. Shall we use labels for that?

**Reply by lwesterhof** [reply]
Yes, I've been using the consortium label for this.

**Comment by Danny-dK** [comment]
What if the Yoda portal cannot be reached? Then the status update on the Yoda portal is not relevant. I'd rather have a more elaborate status update here https://servicedesk.surf.nl/wiki/display/WIKI/Service+status (or at the very least be notified in the portal and aforementioned link) and be notified by email on any maintenance plans. As an institute we would then make sure that our Yoda users get notified by any maintenance.

**Reply by a-surfer** [reply]
@Danny-dK , indeed, relevant case, which I see as a different use case. I would call what you suggest a splash screen. That splash screen is served from something else than the Yoda portal itself (because that one is not working, in your use case). We use it already in other web-offered services at SURF too.

**Comment by peer35** [comment]
This came up in discussions we had about communicating the upcoming 1.9 upgrade and SRAM migration. So we second this idea.

Since not all users read their all their emails having a banner or even being able to alter the content of the "Welcome to Yoda!" box would be a great additional method for announcing planned maintenance or other relevant news.

To add to a-surfer's user story: the content of the banner or "info box" should be configurable in the Yoda portal by an admin. Having the ability to set a start and end date for a message is a bonus.

**Reply by lwesterhof** [reply]
Note that for now customization of the "Welcome to Yoda!" box is already possible using the theme functionality: https://github.com/UtrechtUniversity/yoda-portal/blob/development/themes/vu/index.html

**Reply by a-surfer** [reply]
@peer35 , good idea! I had not thought of the use case of a Yoda admin setting some content from the Yoda portal itself; I had only thought of the feature from the Yoda provider's perspective. I guess the feature could be expanded to accommodate both inputs (i.e.: from the back-end, and from the portal)

**Comment by Danny-dK** [comment]
@a-surfer Would SURF not be able to place banners over the portals of Yoda with expected downtime when maintenance is being performed? (they own the sites). Or is it specifically that there needs to be a maintenance banner option available with the yoda portal for SURF to activate a banner?

**Reply by a-surfer** [reply]
Hey @Danny-dK ,

Interesting, how this issue, which was meant to be a one-off test to see how we could use Ideas in github as a place to catch feature requests, has actually gotten some attention!

Technically everything is possible, indeed. We would like the feature to be part of the product, and not just a one-off tweak. In my proposed suggestion, ideally we can enable content somewhere as part of a configuration, and not directly modify HTML in production (which is what I derive from your comment).

But, yes, I guess the gist of what this Idea portrays is in line with what you are suggesting as a possible solution.

**Reply by Danny-dK** [reply]
@lwesterhof could this be a feature request for next year (or whenever time is available)?

**Reply by Danny-dK** [reply]
part of https://github.com/UtrechtUniversity/yoda/issues/366#issue-2148775291. Discussion could most likely be closed here. @lwesterhof
